app/frontend/components/pages/dashboard/premiumBanner.tsx

```diff
@@ -0,0 +1,31 @@
+import {h} from 'preact'
+import {connect} from '../../../helpers/connect'
+import actions from '../../../actions'
+
+interface Props {
+  closePremiumBanner: () => void
+}
+
+const PremiumBanner = ({closePremiumBanner}: Props) => {
+  const premiumMessage =
+    'VIP STAKING SERVICES - Do you own more than 3 million ADA? Contact us on  adalite@vacuumlabs.com to join our VIP staking services. We will assign you a dedicated account manager and provide you with priority support.'
+  return (
+    <div className="banner premium">
+      <div className="banner-text">{premiumMessage}</div>
+      <button
+        className="button close banner-close"
+        {
+        ...{ariaLabel: 'Close banner'} /* silence ts*/
+        }
```

Comment:
what does this do?

Reply:
Good question, I was just copy-paste-ing this so I have no idea, but it seems the previous version of typescript has some problem with this, and this was a hotfix. I ll remove it from where it's useless. :+1:

Reply:
[This link](https://www.aditus.io/aria/aria-label/#:~:text=Over%2Dusing%20it-,Definition,content%20as%20the%20accessibility%20label.
) explains the usage of this attribute, so Im gonna let it be as it is.

Reply:
I was referencing the style it's written, does 
```
<button
        className="button close banner-close"
        ariaLabel="Close banner"
        onClick={(e) => {
          closePremiumBanner()
        }}
/>
```

not work?

Reply:
Exactly, that doesn't work a that's the reason for that comment.